src/components/PrivateRoute/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+import { useAuth } from 'hooks/Hooks';
+
```

Comment:
group imports like this and sort them alphabetically by their path

react
//space
3rd party imports
//space
in project absolute imports
//space
in project relative imports